Create GifBar.yml

joshghent opened the pull request:
Added project https://github.com/joshghent/gifbar
Great work you're doing here - thank you very much!

joshghent committed:
Create GifBar.yml

Added project https://github.com/joshghent/gifbar

ritwik12 commented on _data/projects/gifbar.yml:
```diff
@@ -0,0 +1,9 @@
+- javascript
+upforgrabs:
+  name: help wanted
+  link: https://github.com/joshghent/gifbar/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22
```
@joshghent Please change this to https://github.com/joshghent/gifbar/labels/help%20wanted

joshghent replied:
Hey, no worries! Just done this, thanks for reviewing 👍

joshghent committed:
Updated up for grabs link to filter on the label

Do not filter out issues that are closed and just use the label text as a filter

ritwik12 commented:
@joshghent Thanks!!
